WordPress/Classes/ViewRelated/Me/MyDomains/MyDomainsViewController.swift

```diff
@@ -0,0 +1,67 @@
+import UIKit
+
+final class MyDomainsViewController: UIViewController {
+    enum Action: String {
```

Comment:
These changes are not intentional right? Perhaps if we merge from trunk they would go away.

Reply:
Yes, that file shouldn't exist. I had some trouble rebasing branches so that file must have slipped through the cracks. Thanks for the catch.